test/test.tf

```diff
@@ -5,6 +5,80 @@ provider "newrelic" {
   region        = var.newrelic_region
 }
 
+provider "pagerduty" {
+  token = var.pagerduty_api_token
+}
+
+data "pagerduty_vendor" "newrelic" {
+  name = "New Relic"
+}
+
+data "pagerduty_user" "dummy" {
```

Comment:
This doesn't create a dummy user. We need a real one. Thus, I wouldn't use "dummy" here. I'd say "user" is better.